hoprd/rest-api/src/alias.rs

```diff
@@ -88,10 +93,11 @@ pub(super) async fn aliases(State(state): State<Arc<InternalState>>) -> impl Int
     }
 }
 
-// TODO(deprecated, will be removed in v3.0) Get each previously set alias and its corresponding ETH address as a hashmap.
+/// (deprecated, will be removed in v3.0) Get each previously set alias and its corresponding ETH address as a hashmap.
```

Comment:
Could you also address this, perhaps in a separate PR? Is there and issue?

Reply:
Yes, including the DB. 🍻